appinfo/info.xml

```diff
@@ -8,7 +8,7 @@
 	<licence>AGPL</licence>
 	<author>Roeland Jago Douma</author>
 	<default_enable/>
-	<version>1.0.0</version>
+	<version>1.0.1</version>
```

Comment:
Doesn't mean I'll just release this as 1.0.1, but one gotta start at some point to increase the number :-)